src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -766,18 +791,34 @@ private Task WriteNonPersonalizedFilesToVmAsync(ConnectionInfo sshConnectionInfo
                 () => UploadFilesToVirtualMachineAsync(
                     sshConnectionInfo,
                     new[] {
-                        (Utility.GetFileContent("scripts", "startup.sh"), $"{CromwellAzureRootDir}/startup.sh", true),
+                        // startup.sh must be written during update and doesn't have specific configuration file needs.
+                        (Utility.PersonalizeContent(new[]
+                        {
+                            new Utility.ConfigReplaceTextItem("{ReplaceMySqlSha}", configuration.ProvisionMySQLOnAzure.GetValueOrDefault() ? String.Empty : "kv[\"MySqlImageSha\"]=\"\""),
```

Comment:
I would have preferred replacing the previous content with an empty line in the `true` case instead of inserting the previous content in the `false` case. Ditto for the next line and for `ReplaceMySqlLocalOrAzure`

Reply:
got it and changed ReplaceMySqlLocalOrAzure to regex 👍